lib/auth/auth.go

```diff
@@ -2776,6 +2776,8 @@ func (a *Server) CreateWebSession(ctx context.Context, user string) (types.WebSe
 }
 
 // GenerateToken generates multi-purpose authentication token.
+// Deprecated: REMOVE IN 14.0.0 - Clients should now use CreateToken or
+// UpdateToken.
 func (a *Server) GenerateToken(ctx context.Context, req *proto.GenerateTokenRequest) (string, error) {
```

Comment:
Would it be possible to replace GenerateToken with Create or UpdateToken, while keeping the test scenarios? It could make some of the future cleanups smaller/simpler.